### DISCUSSION TITLE
How to add this plugin to grafana cloud

### DISCUSSION THREAD

**Discussion by skngetich** [grafana/grafana-infinity-datasource#70] [Q&A] [answered]
How can I install this plugin in the grafana cloud.I could not get it in the plugin section

**Comment by yesoreyeram** [comment, marked as answer]
Infinity plugin is now available in [grafana.com plugins page](https://grafana.com/grafana/plugins/yesoreyeram-infinity-datasource/). You can install from there into your instance of grafana.

**Reply by skngetich** [reply]
Thank you let me try it out
